contributing/DOCS-CONTRIBUTING.md

```diff
@@ -68,45 +71,203 @@ documentation efforts, who are happy to help you get started.
 
 There are a couple different ways to jump in to the Knative doc set:
 
-- Look at issues in the Docs repo labled
-  [Good First Issue](https://github.com/knative/docs/labels/kind%2Fgood-first-issue).
-- Run through the [install guide](../install/README.md) for the platform of your
-  choice, as well as the
-  [Getting Started with Knative App Deployment](../install/getting-started-knative-app.md)
-  guide, and keep a
-  [friction log](https://devrel.net/developer-experience/an-introduction-to-friction-logging)
-  of the experience. What was hard for you? Then open a PR with a few
-  improvements you could make from the friction log you kept. How can you make
-  the documentation better so that others don't run into the same issues you
-  did?
+- Look for a
+  [Good First Issue](https://github.com/knative/docs/labels/kind%2Fgood-first-issue)
+  in the backlog.
 
-### Submitting documentation PRs - what to expect
+- Try out Knative and send us feedback. For example, run through one of the
+  [install guides](../install/README.md) and then try
+  [Getting Started with Knative Serving](../install/getting-started-knative-app.md).
+
+    You should keep a
+    [friction log](https://devrel.net/developer-experience/an-introduction-to-friction-logging)
+    of your experience and then use that to open your first set of PRs.
+    Examples:
+
+   - What was hard for you?
+   - Did you stumble on something because it wasn't clear?
+   - Was a dependency not mentioned?
+
+**A few pointers and other considerations**
+
+  - Think about how you can improve the documentation as a whole. <br>
+    For example, how can you fix the issue you found so that others don't run
+    into the same challenges?
 
-Here's what will happen after you open a PR in the Docs repo:
+  - Are there multiple places that could be fixed?
 
-1. One of the Docs approvers will triage the PR and provide an initial
-   documentation review as soon as possible.
-   1. If the PR involves significant changes or additions to sample code, we'll
-      flag it for engineer review.
-1. Once we're satisfied with the quality of the writing and the accuracy of the
-   content, we'll approve the change.
+     - Are there other pages that you should apply your update?
+
+     - Would it help if there was a link to more or related information on a
+       the page? On a related page?
+
+  - Was the title or description misleading? Did you expect to find something
+    else?
+
+  - If you find something and dont have the bandwidth to open an actual PR, help
+    make us aware of the pain point and open an
```

Comment:
```suggestion
    make us aware of the pain point and open an
```

Reply:
something odds going on here, nothing shows as changed, marking this resolved